setup.py

```diff
@@ -50,8 +50,8 @@ def runSetup():
                 'gcs_oauth2_boto_plugin==1.9',
                 botoRequirement],
             'cwl': [
-                'cwltool==1.0.20170413194156',
-                'schema-salad==2.5.20170328195758',
+                'cwltool>=1.0.20170811195303',
+                'schema-salad>=2.6.20170806163416',
```

Comment:
I also wonder why, conversely, `'cwltest>=1.0.20170214185319'` is not pinned that strictly?